homeassistant/components/ezviz/strings.json

```diff
@@ -71,5 +71,15 @@
         }
       }
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "alarm_schedules_enabled": {
+        "name": "Alarm schedules"
```

Comment:
Wouldn't Alarm schedules enabled be a better name? I don't own any ezviz devices so maybe it's a recognizable name when you have them, but if this is a common name, then it's fine

Reply:
It would yes, I'll change it.